configs/PyTorchLightning/flash_pl-develop.yaml

```diff
@@ -1,9 +1,9 @@
 target_repository:
   HTTPS: https://github.com/PyTorchLightning/lightning-flash.git
   # checkout a particular branch or a tag
-  checkout: master
+  checkout: ecosystem
```

Comment:
mind add note why this branch?

Reply:
Just testing, was having some failures so trying to debug, will be switched back once done 😃